plugins/navigation/src/main/resources/NavigationSpeech.aim.xml

```diff
@@ -161,27 +161,33 @@
 				<gram>({endregistry}|{endwildcard})</gram>
 			</EntityTemplate>
 			<EntityTemplate id="mode" required="true">
-				<gram>(car|transit|bike|bicycle)</gram>
+				<gram>(car|[public] (transport|transit)|bike|bicycle)</gram>
+			</EntityTemplate>
+			<EntityTemplate id="time" required="true">
+				<gram>({amytime}|{amydatetime})</gram>
 			</EntityTemplate>
 		</EntityTemplates>
 		<Prompt entityTemplateId="start">
 			<text>from which location?</text>
 			<gram>[from] {start}</gram>
 		</Prompt>
-
+		<Prompt entityTemplateId="time">
+			<text>When are you leaving?</text>
```

Comment:
If this senctence starts with an uppercase letter, every other sentence should do that too